api/common/id.go

```diff
@@ -17,4 +17,9 @@
 
 package common
 
-type ChunkID uint64
+type ChunkID []byte
+type SeriesID uint64
```

Comment:
What's the purpose of the `SeriesID`?

Reply:
It's the routing key that is defined in `sw/trace_series.json`. Liaison leverages it to determine which shard a segment should be forward.